src/category_theory/localization/predicate.lean

```diff
@@ -41,4 +49,132 @@ instance Q_is_localization : W.Q.is_localization W :=
 
 end functor
 
+namespace localization
+
+/-- This universal property states that a functor `L : C ⥤ D` inverts morphisms
+in `W` and the all functors `D ⥤ E` (for a fixed category `E`) uniquely factors
+through `L`. -/
+@[nolint has_nonempty_instance]
```

Comment:
You provide a "canonical" example below, so I think you can just provide an `inhabited` instance.